packages/finance/Scarb.toml

```diff
@@ -1,16 +1,22 @@
 [package]
 name = "openzeppelin_finance"
+readme = "README.md"
+keywords = [
+    "openzeppelin",
+    "starknet",
+    "contracts",
+    "finance",
+    "vesting"
```

Comment:
```suggestion
    "finance"
```

Reply:
Not that I'm strongly disagree, but could you explain the reason behind removing `vesting` keyword? Won't it make the package more discoverable, especially while it's only purpose is providing Vesting component?

Reply:
Good points, I think we should keep it. I didn't have a strong opinion on removing it either, it just felt a bit too specific for the package, but I think it may be helpful.